Apps/phironnet/ironnet.json

```diff
@@ -1290,6 +1290,20 @@
                         "https://1.2.3.4/event?uuid=9f61e9e9af904540ac8bd730f5ed48d2"
                     ]
                 },
+                {
+                    "data_path": "action_result.data.*.events.*.src_network_id",
+                    "data_type": "string",
+                    "example_values": [
+                        "skynet-12345"
```

Comment:
Lets make the sample network ID `sample-network`.

Reply:
fixed